File: Jour11/jour11.py
#coding : utf-8
while 1:
	n = input("Donner un entier superieur ou egal a 1 : ")
	try:
		n = int(n)
		if(n<1):
			print("Saisie invalide !")
			continue
	except:
		print("Saisie invalide !")
		continue
	else:
		break
for i in range(n):
	print(i+1)

File: Jour11/jour11.c
#include<stdio.h>

int main()
{
	int n,i;
	do
	{
		printf("Donner un entier superieur ou egal a 1 : ");
		scanf("%d",&n);
		if (n<1)
			printf("Votre saisie est incorrect !\n");
	}while(n<1);
	for(i = 1; i<= n; i++)
		printf("%d\n",i);
	
	return 0;
}